vulnfeeds/cpp/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+cd ../..
```

Comment:
I think this should just be `cd ..` if I'm following the directory structure right.

Reply:
Good catch, thank you. Corrected in [c1dcd02](https://github.com/google/osv.dev/pull/1306/commits/c1dcd02d5aeeaedfbe9a47719bdcf0517287064e)